recipes/distro/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "distro" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "722054925f339a39ca411a8c7079f390a41d42c422697bedf228f1a9c46ac1ee" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  entry_points:  # [linux]
```

Comment:
You don't want to put a linux selector on `entry_points` or `test`. If this package is linux only, you can add under `build`:

```
build:
   skip: True  # [not linux]
```

Reply:
Also if you're just building on linux, you could add ` [skip appveyor]` to the commit message as well as `[skip travis]` for osx

Reply:
Allright. I think this package is intended for linux only.